metricq/agent.py

```diff
@@ -184,6 +184,17 @@ def __init__(
             __version__,
         )
 
+    @property
+    def url(self) -> str:
+        """
+        Redacts login information from the management URL.
```

Comment:
That's a description of what the method does to the management URL, not what it returns.